druid/src/window.rs

```diff
@@ -226,11 +227,15 @@ impl<T: Data> Window<T> {
                 is_root: true,
             };
 
+            let _span = info_span!("event");
+            let _span = _span.enter();
             self.root.event(&mut ctx, &event, data, env);
+            std::mem::drop(_span);
```

Comment:
unstead of doing this, could you try using scoping to drop the value? I am not sure if thats how it works in rust but it would be cleaner and lesshacky since a span over a scope makes more sense logically (imo)

Reply:
I think the proper syntax is this:

```rust
let span = info_span!("my span");
span.in_scope(|| {
    // do work inside the span...
});
```

I have no idea about this info_span thingie, but that's from tracing docs.

Reply:
I'll look into scoping, but the closure-based solution doesn't work because of the borrow checker.

Reply:
Done.